go.mod

```diff
@@ -2,17 +2,19 @@ module github.com/longhorn/longhorn-instance-manager
 
 go 1.21
 
+replace github.com/longhorn/go-common-libs v0.0.0-20240219094750-e7176c332156 => github.com/ChanYiLin/go-common-libs v0.0.0-20240305065347-bd2b05c805a6
+
 require (
 	github.com/RoaringBitmap/roaring v1.9.0
 	github.com/google/uuid v1.6.0
-	github.com/longhorn/backupstore v0.0.0-20240126140439-9c90a8728ca2
-	github.com/longhorn/go-common-libs v0.0.0-20240109042507-23627e6416b7
-	github.com/longhorn/go-spdk-helper v0.0.0-20240117135122-26f8acb2a13d
+	github.com/longhorn/backupstore v0.0.0-20240219094812-3a87ee02df77
+	github.com/longhorn/go-common-libs v0.0.0-20240219094750-e7176c332156
+	github.com/longhorn/go-spdk-helper v0.0.0-20240301101140-6eb6aa5fc09d
 	github.com/longhorn/longhorn-engine v1.6.0-dev-20240105.0.20240126141003-067f67803ee8
-	github.com/longhorn/longhorn-spdk-engine v0.0.0-20240123044045-c5f14845bd83
+	github.com/longhorn/longhorn-spdk-engine v0.0.0-20240226075747-9b8228c933d5
```

Comment:
Don't update `longhorn-spdk-engine` dep in this PR, so you should be able to move this PR forwards. Daminao will take care of that update in another PR.

Reply:
cc @DamiaSan